docs/memory/memory-qos.md

```diff
@@ -0,0 +1,153 @@
+# Memory QoS NRI plugin
+
+This NRI plugin adds two methods for controlling cgroups v2 `memory.*`
+parameters: QoS class and direct memory annotations.
+
+## Workload configuration
+
+There are two configuration methods:
+1. Memory QoS classes: memory parameters are calculated in the same
+   way for all workloads that belong to the same class.
+2. Direct workload-specific memory parameters.
+
+Memory QoS class of a pod or a container is defined using annotations
+in pod yaml:
+```
```

Comment:
add empty line plus the format hint

```suggestion

```yaml
```

Reply:
Fixed.